server/src/main/java/io/crate/metadata/pgcatalog/PgCatalogTableDefinitions.java

```diff
@@ -197,6 +197,19 @@ public PgCatalogTableDefinitions(InformationSchemaIterables informationSchemaIte
             (user, p) -> p.owner().equals(user.name()),
             PgSubscriptionRelTable.create().expressions()
         ));
+
+        tableDefinitions.put(PgTablesTable.IDENT, new StaticTableDefinition<>(
+            informationSchemaIterables::relations,
```

Comment:
`informationSchemaIterables::relations` will contain both, tables and views, see https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/engine/collect/sources/InformationSchemaIterables.java#L119.
Please see my previous suggestion on how to list tables only.

Reply:
I'd suggest to have test(s) to verify that, to select a table from `pg_views` and a view from `pg_tables`.

Reply:
Oops, sorry 🙈 

Would it then not be more consistent to also have an Iterable for tables?

Reply:
> Would it then not be more consistent to also have an Iterable for tables?

Also fine for me.

Reply:
I added the tables Iterable